LICENSE.txt

```diff
@@ -10,7 +10,7 @@ modification, are permitted provided that the following conditions are met:
   b. Redistributions in binary form must reproduce the above copyright
      notice, this list of conditions and the following disclaimer in the
      documentation and/or other materials provided with the distribution.
-  c. Neither the name of toolz nor the names of its contributors
+  c. Neither the name of copyright holder nor the names of its contributors
```

Comment:
Nice catch @aregm, and thank you for fixing!

I'm wondering why the proposed change is to use `copyright holder` instead of just swapping out `toolz` for `zict` 

```suggestion
  c. Neither the name of zict nor the names of its contributors
```

Reply:
I just copy/pasted the generic BSD 3 clause. As soon as you customize it, the legal license checking system will tag it as a custom BSD-3, and in enterprises, it requires more bureaucracy. But I don't mind. Up to you guys.

Reply:
Ah, I see -- thank you for clarifying. Looking at the Dask license (which is BSD 3) we're missing the work "the" here. Otherwise this looks good to go 

```suggestion
  c. Neither the name of the copyright holder nor the names of its contributors
```